code/game/machinery/suit_storage_unit.dm

```diff
@@ -60,6 +60,10 @@
 	storage_type = /obj/item/weapon/watertank/atmos
 
 /obj/machinery/suit_storage_unit/mining
+	suit_type = /obj/item/clothing/suit/hooded/explorer
+	mask_type = /obj/item/clothing/mask/gas
+
+/obj/machinery/suit_storage_unit/mining/eva

```

Comment:
LIES. Mine type only affects some of the map generation stuff. You had to manually replace lavaland with z5 and re-enable crosslinking at least.